Xamarin.Forms.Core/Internals/IndicatorViewExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.ComponentModel;
+using static Xamarin.Forms.IndicatorView;
+
+namespace Xamarin.Forms.Internals
+{
+	[EditorBrowsable(EditorBrowsableState.Never)]
+	public static class IndicatorViewExtensions
+	{
+		public static void SetItemsSourceBy(this IndicatorView indicatorView, CarouselView carouselView)
```

Comment:
is this meant to be used ?

Reply:
in another case, we'll have to bind each property from indicators to carousel